pages/common/ansible-inventory.md

```diff
@@ -0,0 +1,21 @@
+# ansible-inventory
+
+> Display or dump an Ansible inventory.
+> See also: `ansible`.
+> More information: <https://www.ansible.com/>.
+
+- Display the default inventory:
+
+`ansible-inventory --list`
+
+- Display a custom inventory:
+
+`ansbile-inventory --list --inventory {{path/to/inventory}}`
```

Comment:
Is this correct?
```suggestion
`ansbile-inventory --list --inventory {{path/to/inventory_file_or_dirctory}}`
```

Reply:
An inventory can be a directory. It can also be a script. Would it better to use "inventory_file_or_script_or_directory"?

Reply:
That seems a little long, but probably fine.